packages/react-is/index.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+'use strict';
+
+const ReactIs = require('./src/ReactIs');
+
+// TODO: decide on the top-level export form.
+// This is hacky but makes it work with both Rollup and Jest.
+module.exports = ReactIs.default ? ReactIs.default : ReactIs;
```

Comment:
We should do this correctly right away since it's not a legacy package.

Let's remove the default export here and only provide named exports. That's how we make it work with tree shaking, too (when we provide an ES build output).

Reply:
To clarify, I'm saying this file shouldn't need to use CommonJS at all. Just re-export `*` from the source.

Reply:
Hm. Okay.

Why didn't we follow that pattern with other, brand-new packages (like `react-reconciler`)?

Reply:
I think there was some reason but I don't remember straight away.

It's not very important in case of the reconciler though because it provides just one function. `react-is`, on the other hand, provides a bunch, and is expected to grow.

Call/Return does avoid the `.default` hack: https://github.com/facebook/react/blob/29e8924c70856bef9b11e0c74a450140bfcce773/packages/react-call-return/index.js#L12

If there's some issue with top-level ES import (maybe there was) then the above is sufficient.

Reply:
Okay. This makes sense. Thanks for the suggestion. 😄

It's been updated.